.yamato/project-pack.yml

```diff
@@ -0,0 +1,17 @@
+{% metadata_file .yamato/metadata.metafile %}
+---
+pack:
+  name: Pack project
+  agent:
+    type: Unity::VM
+    image: package-ci/ubuntu:stable
+    flavor: b1.large
+  commands:
+    - npm install upm-ci-utils@stable -g --registry https://artifactory.prd.cds.internal.unity3d.com/artifactory/api/npm/upm-npm
+    - upm-ci project pack --project-path Projects/URP
```

Comment:
My understanding is that the 3 project pack commands are doing exactly the same thing in this case. It's packing package. (see upm-ci read-me)

Reply:
Fixed in latests commits